cmd/aad-cli/cli/user_test.go

```diff
@@ -102,13 +104,21 @@ func TestUser(t *testing.T) {
 				username := args[slices.Index(args, "--name")+1]
 				user, err := cache.GetUserByName(context.Background(), username)
 				require.NoError(t, err, "Setup: failed to get user from cache")
-				// Timestamps get serialized as RFC3339 which includes timezone
-				// information.
-				// We replace this with the unix timestamp to make
-				// the comparison easier.
-				got = testutils.TimestampToUnix(t, got, user.LastOnlineAuth)
-			}
 
+				if slices.Contains(args, "last_online_auth") {
+					i, err := strconv.ParseInt(strings.Trim(got, "\n"), 10, 64)
+					require.NoError(t, err, "Expected no error but got one")
+					require.Equal(t, user.LastOnlineAuth.Unix(), i)
+					return
+				}
+
+				if len(args) < 4 {
```

Comment:
I guess this is for `get_user*`. A little bit like the previous one, I’m not a big fan of not having golden files for those use case and having this only checked in the tests (this test now is a mix of golden and non golden file). Let’s see what @GabrielNagy think.

Reply:
I agree that it would be better to print the values in the same way. And it does make our life easier as well, I'll do that.